packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js

```diff
@@ -113,20 +144,59 @@ export const TableRows = ({
                 />
               </Td>
             )}
+
             {headers.map(({ key, cellFormatter, name, ...rest }) => {
+              if (hasDraftAndPublish && name === 'publishedAt') {
+                return (
+                  <Td key={key}>
+                    <Status
+                      width="min-content"
+                      showBullet={false}
+                      variant={data.publishedAt ? 'success' : 'secondary'}
+                      size="S"
+                    >
+                      <Typography
+                        fontWeight="bold"
+                        textColor={`${data.publishedAt ? 'success' : 'secondary'}700`}
+                      >
+                        {formatMessage({
+                          id: getTrad(
+                            `containers.List.${data.publishedAt ? 'published' : 'draft'}`
+                          ),
+                          defaultMessage: data.publishedAt ? 'Published' : 'Draft',
+                        })}
+                      </Typography>
+                    </Status>
+                  </Td>
+                );
+              }
+
+              if (hasReviewWorkflows && name === 'strapi_reviewWorkflows_stage') {
+                return (
+                  <Td key={key}>
+                    {data.strapi_reviewWorkflows_stage ? (
+                      <ReviewWorkflowsStage
+                        color={
+                          data.strapi_reviewWorkflows_stage.color ?? lightTheme.colors.primary600
```

Comment:
i'd have applied this default in the component to be honest. You can also just do the string `"primary600"` i think